zigpy/zgp/message_concentrator.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Ah, sorry. The HA version I was testing against required 3.11, which I believe has annotations by default.